tap-agent/src/agent/sender_accounts_manager.rs

```diff
@@ -483,7 +483,7 @@ async fn new_receipts_watcher(
             );
         }
         RECEIPTS_PER_SENDER_ALLOCATION
-            .with_label_values(&[&sender_allocation_str])
+            .with_label_values(&[&sender_address.to_string(), &allocation_str])
```

Comment:
You can use `&allocation_id.to_string()` instead of `&allocation_str`

Reply:
Did it that way due to:
![image](https://github.com/graphprotocol/indexer-rs/assets/131038589/b37f81bf-df74-48d6-a582-add7568a009c)

Reply:
Using that way because of borrow checker.